dev-docs/guides/fees.md

```diff
@@ -104,6 +104,6 @@ At times, we may want to waive fees on revenue. For example, if the transaction
 
 There are two ways of waiving fees, set `fee_waived` on the `CanonicalPendingTransaction` or set `reason` on the `Fee` to “TBD”.
 
-Occasionally, we make this waiver after charging a fee. This results in a fee credit. Which is essentially a fee where we (HCB) pay the organisation instead of them paying us.
+Occasionally, we make this waiver after charging a fee. This results in a fee credit. A free credit is essentially a fee where we (HCB) pay the organisation instead of them paying us.
```

Comment:
```suggestion
Occasionally, we make this waiver after charging a fee. This results in a fee credit. A fee credit is essentially a fee where we (HCB) pay the organisation instead of them paying us.
```

Reply:
(i think this is mb - sorry)